books/free-programming-books-langs.md

```diff
@@ -1086,7 +1087,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 ### JavaScript
 
-* [Airbnb JavaScript Style Guide() {](https://airbnb.io/javascript/) - Airbnb (HTML)
```

Comment:
is this a typo?

Reply:
Yeah. The title of the page referred to in the link doesn't have { or (). So I removed them.